.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -338,7 +348,7 @@ void gradleVersionsUpdate(String repo, String newVersion) {
 
 void archiveJUnitTestResults() {
     if (!skipUnitTests()) {
-        junit testResults: '**/target/surefire-reports/**/*.xml, **/target/failsafe-reports/**/*.xml', allowEmptyResults: false
+        junit testResults: '**/target/surefire-reports/**/*.xml, **/target/failsafe-reports/**/*.xml', allowEmptyResults: true
```

Comment:
allowing empty results as if the build fails, then there is no test results and this will also fail ...